base/array.jl

```diff
@@ -974,14 +978,29 @@ end
 
 Insert the elements of `items` to the beginning of `a`.
 
+**See also:**
+[`append!`](@ref), [`insert!`](@ref)
+
 # Examples
 ```jldoctest
 julia> prepend!([3],[1,2])
 3-element Array{Int64,1}:
  1
  2
  3
+
+julia> prepend!([1, 2, 3], [4, 5, 6])
+6-element Array{Int64,1}:
+ 4
+ 5
+ 6
+ 1
+ 2
+ 3
 ```
+
+Use [`unshift!`](@ref) to add individual items to `collection`. The result of
```

Comment:
I'd rather put this in the "See also" paragraph, rather than after the examples, where people might not find it (examples can be long). No need to refer to a particular example, just say "`prepend!(x, y)` is equivalent to...".

Reply:
I removed these sentences in favor of a "See also:" entry. We opted against further descriptions in "See also" earlier.